.github/ISSUE_TEMPLATE/1_Bug_report.md

```diff
@@ -1,11 +1,11 @@
 ---
 name: "🐛 Bug Report"
-about: Report a general framework issue
+about: 'Report a general framework issue. Please first check if your Laravel version is still supported: https://laravel.com/docs/5.8/releases#support-policy'
```

Comment:
Always use the latest version:

```suggestion
about: 'Report a general framework issue. Please first check if your Laravel version is still supported: https://laravel.com/docs/releases#support-policy'
```

Reply:
The docs don't support page forwards with anchor tags.

Reply:
Hmm, for some reason this works now. weird

Reply:
Pushed. Thanks!